src/app/models.py

```diff
@@ -121,3 +121,42 @@ def __init__(self, login, password, user_id, url, title, comment):
         self.url = url
         self.title = title
         self.comment = comment
+
+
+class SessionObject(Base):
+    __tablename__ = 'session_objects'
+
+    id = Column('id', Integer, primary_key=True)
+    token = Column('token', String(100), unique=True, nullable=False)
+
+    user_id = Column('user_id', Integer, ForeignKey('users.id'))
+    user = relationship("User", backref="session_objects", cascade='all,delete')
+
+    login_time = Column('login_time', Date, nullable=False)
```

Comment:
Date type not valid for a timeout check. What if your session is valid for 30 minutes?